en/advanced/ai.md

```diff
@@ -0,0 +1,79 @@
+# AI functionality in JabRef
+
+## Chat model
+
+**Type**: enumeration
```

Comment:
I think, this is a computer-science term. "List" is more user-friendly, isn't it?